### PR TITLE
refactor(neuron-ui): refactor wallet setting

### PR DESCRIPTION
**Pull request by Keith-CY**
1. remove pagination in wallet setting
2. add scroll list in wallet setting and network setting
3. group hooks in wallet setting

### CODE REVIEW

File: packages/neuron-ui/src/components/WalletSetting/index.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          {walletToDelete ? (
+            <Card.Header>{t('settings.wallet-manager.delete-wallet-title', { name: walletToDelete.name })}</Card.Header>
+          ) : (
+            <Card.Header>Not found</Card.Header>
+          )}
+          {walletToDelete ? (
+            <Card.Body>
+              <Form.Group as={Row} controlId="formPlaintextPassword">
+                <Col>
+                  <Form.Control type="password" placeholder="password" onChange={deleteWallet.password.onChange} />
```
Comment (ashchan) [inline packages/neuron-ui/src/components/WalletSetting/index.tsx]: Placeholder text should be localized with i18n as well.

Comment (Keith-CY) [reply packages/neuron-ui/src/components/WalletSetting/index.tsx]: updated in https://github.com/nervosnetwork/neuron/pull/263/commits/7889c7b9e2706cf59952fbd6ae153570d754f634

**Comment by ashchan** [comment]
Please check CI status

**Comment by Keith-CY** [comment]
CI passed, and I found that the linter in CRA3.0 seems not to work well, maybe need more optimization.

**Comment by ashchan** [comment]
> CI passed, and I found that the linter in CRA3.0 seems not to work well, maybe need more optimization.

Not surprising regarding the linting thing ;-) Can you take care of that? (but not urgent)

**Pull request merged**